sys/riscv/conf/CHERI-PURECAP-FETT

```diff
@@ -0,0 +1,9 @@
+#NO_UNIVERSE
+
+include "FETT"
+include "std.CHERI-PURECAP"
+
+options 	ENABLE_PAST_LOCAL_VULNERABILITIES
+options 	ENABLE_PAST_REMOTE_VULNERABILITIES
```

Comment:
These don't make sense to commit without the corresponding vulnerabilities and config options available.